test/TestRun.hs

```diff
@@ -104,11 +110,15 @@ runTest path flags = do
       normalise [] = []
 
 main :: IO ()
-main =
-  defaultMainWithIngredients ingredients $
-    askOption $ \(NodeOpt node) ->
-      let (codegen, flags) = if node then (JS, ["--codegen", "node"])
-                                     else (C , [])
-       in
-        mkGoldenTests (testFamiliesForCodegen codegen)
-                    (flags ++ idrisFlags)
+main = do
+  -- The Node.js tests are nondeterministically wrong when run in parallel for
+  -- some reason, so we turn off parallelism when --node is passed. There's no
+  -- way to change paralellism after the test runner has started, so we parse
```

Comment:
Sorry to be pedantic, however, parallelism has been spelled incorrectly in its second use.